aws_lambda_powertools/utilities/data_classes/vpc_lattice.py

```diff
@@ -169,16 +169,6 @@ def version(self) -> str:
         """The VPC Lattice v2 Event version"""
         return self["version"]
 
-    @property
-    def is_base64_encoded(self) -> bool | None:
-        """A boolean flag to indicate if the applicable request payload is Base64-encode"""
-        return self.get("isBase64Encoded")
-
-    @property
-    def path(self) -> str:
-        """The VPC Lattice v2 Event path"""
-        return self["path"]
-
```

Comment:
Please add a comment saying that `is_base64_encoded` and `path` are inherited from `BaseProxyEvent` class.